app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -2,9 +2,7 @@ package com.kickstarter.libs
 
 import com.kickstarter.libs.KoalaContext.*
 import com.kickstarter.libs.KoalaEvent.ProjectAction
-import com.kickstarter.libs.utils.BooleanUtils
-import com.kickstarter.libs.utils.ExperimentData
-import com.kickstarter.libs.utils.AnalyticEventsUtils
+import com.kickstarter.libs.utils.*
```

Comment:
Sorry I didn't notice this before. We avoid using star-imports on new code. If you could revert these changes and add whatever specific utils classes you need, that would be great. Thanks!